recipes/osx-sysroot/conda_build_config.yaml

```diff
@@ -0,0 +1,16 @@
+sdk_version:
+  - "10.9"
+  - "10.10"
+  - "10.11"
+  - "10.12"
+  - "10.13"
+  - "10.14"
+  - "10.15"
+  - "11.0"
+  - "11.1"
+  - "11.3"
```

Comment:
What did you base this list on? AFAIU after 11.0, the minor version is irrelevant. However, there's 12 & 13 live already.

Reply:
I google randomly. Will look more.